src/components/TextInput/index.jsx

```diff
@@ -1,7 +1,46 @@
-import React from "react";
+import React, { useState } from "react";
+import styles from "./styles/styles.module.scss";
 
-const TextInput = () => {
-  return <p>text input</p>;
+const TextInput = ({ label, minlength, isRequired = true }) => {
+  // let [inputStyle, setInputStyle] = useState(styles.input);
+  let [errorText, setErrorText] = useState("");
+
+  const handleInputValue = (event) => {
+    if (event.target.value.length === 0) {
+      setErrorText("can't be empty");
```

Comment:
First word in a sentence is always uppercase.
```suggestion
      setErrorText("Can't be empty.");
```

Reply:
done